functions/src/api/auth/assertValidToken.spec.ts

```diff
@@ -0,0 +1,46 @@
+// tslint:disable-next-line
+import '../../testUtils/mockFirebaseAdminAndFunctions'
```

Comment:
why the need to disable linting here ?

Reply:
side effect related to TSLINT because of the generic import

Reply:
would the lint error persist if writing the import as `import * as mockFirebaseAdminAndFunctions from ...` ?

Reply:
another one by importing an unused module in the file, TS is very picky!